benches/aead.rs

```diff
@@ -0,0 +1,162 @@
+#![allow(missing_docs)]
+
+use criterion::criterion_main;
+use ring::{aead, error};
+
+// All the AEADs we're testing use 96-bit nonces.
+pub const NONCE: [u8; 96 / 8] = [0u8; 96 / 8];
+
+// A TLS 1.2 finished message is always 12 bytes long.
+const TLS12_FINISHED_LEN: usize = 12;
+
+// A TLS 1.3 finished message is "[t]he size of the HMAC output for the
+// Hash used for the handshake," which is usually SHA-256.
+const TLS13_FINISHED_LEN: usize = 32;
+
+// In TLS 1.2, 13 bytes of additional data are used for AEAD cipher suites.
+const TLS12_AD: [u8; 13] = [
+    23, // Type: application_data
+    3, 3, // Version = TLS 1.2.
+    0x12, 0x34, // Length = 0x1234.
+    0, 0, 0, 0, 0, 0, 0, 1, // Record #1
+];
+
+// In TLS 1.3, no additional data is used for AEAD cipher suites.
+const TLS13_AD: [u8; 0] = [];
```

Comment:
At the time I wrote the benchmarks in crypto-bench, the TLS 1.3 AAD was empty. But now it is 5 bytes long in the final RFC. So let's change this to `[u8; 5]` and update the comment.

Reply:
I've updated this to be 5 bytes and changed the comment to go along with it.